pkg/vulnsrc/vulnsrc.go

```diff
@@ -119,11 +119,14 @@ func (u Updater) Update(targets []string) error {
 		}
 	}
 
+	timeUpdatedAt := u.clock.Now().UTC()
+	timeNextUpdate := timeUpdatedAt.Add(u.updateInterval)
+
 	err := u.dbc.SetMetadata(db.Metadata{
 		Version:    db.SchemaVersion,
 		Type:       u.dbType,
-		NextUpdate: u.clock.Now().UTC().Add(u.updateInterval),
-		UpdatedAt:  u.clock.Now().UTC(),
+		NextUpdate: timeNextUpdate,
+		UpdatedAt:  timeUpdatedAt,
```

Comment:
nit: IMHO, if the meaning of the value is difficult to understand, we should define another variable for ease of understanding. For example,

```
return timeUpdatedAt.Add(u.updateInterval)
```
is a bit unclear, so in this case, I prefer the following.

```
timeNextUpdate := timeUpdatedAt.Add(u.updateInterval)
return timeNextUpdate
```



In this case, I feel it looks the same as the following.
```suggestion
		NextUpdate: u.clock.Now().UTC().Add(u.updateInterval),
		UpdatedAt: u.clock.Now().UTC(),
```

I'm guessing you changed here to get pointers of these values and forgot to change it back, but let me know if you have any reason.

Reply:
Yeah I understand. Although doing a `clock.Now()` at two different places will lead to two different values of "now" since when things were about to get updated. In this particular case it's not a big deal. 

Yes I forgot to remove this after the switchback from pointers! Thanks for catching it.